src/domain/certificates/useCases/generateRevocationReason.js

```diff
@@ -0,0 +1,11 @@
+const REVOCATION_LANGUAGE = {
+  PRE_REASON: 'Reason given:',
+  REVOCATION: 'This certificate has been revoked by the issuer.'
+};
+
+export default function generateRevocationReason (reason) {
+  reason = reason.trim();
+  // TODO take strings out to constants
```

Comment:
Lingering TODO, do you want to leave it?

Reply:
Oh right that should be gone